crates/rattler_package_streaming/src/write.rs

```diff
@@ -195,19 +208,23 @@ pub fn write_conda_package<W: Write + Seek>(
 
     let (info_paths, other_paths) = sort_paths(paths, base_path);
 
-    outer_archive.start_file(format!("pkg-{out_name}.tar.zst"), options)?;
+    let archive_path = format!("pkg-{out_name}.tar.zst");
```

Comment:
maybe call it just `.tar` or/and use `tempfile` here?

Reply:
done in 221873d3d27b7b67349a4860c6e0ac2955a0d6eb